src/lib/core/channel-value/channel-value.component.ts

```diff
@@ -81,4 +89,12 @@ export class ChannelValueComponent implements OnChanges {
             ? this.prefixUnitWhitelist.includes(this.units)
             : this.suffixUnitWhitelist.includes(this.units);
     }
+
+    /** Update the text of the Channel value element in the DOM, creating the element if necessary. */
+    private _updateRenderedValue(newValue: string | number): void {
+        const newValueNormalized: any = `${newValue ?? ''}`.trim();
+
+        // Check the value as interger and convert into the string value
+        this._value = !isNaN(Number(newValueNormalized)) ? String(newValueNormalized) : newValueNormalized;
+    }
```

Comment:
This feels like overkill. I would have gone with a simpler approach where you render `{{ getValueAsString() }}` instead of `{{ value }}` in the component template and then use a simple function like this to return the value as a string: 

`getValueAsString(): string {
        return String(this.value);
    }`

If we wanted to make it generic we could take in a value for the getValueAsString function and convert that instead. Feel free to state your case on why your approach is preferred.